src/main/java/be/seeseemelk/mockbukkit/WorldBorderMock.java

```diff
@@ -0,0 +1,136 @@
+package be.seeseemelk.mockbukkit;
+
+import org.apache.commons.lang.Validate;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.WorldBorder;
+import org.bukkit.util.BoundingBox;
+import org.jetbrains.annotations.NotNull;
+
+public class WorldBorderMock implements WorldBorder
+{
+
+	World world;
+	double size;
+	double damageAmount = 0.2;
+	double damageBuffer = 5.0;
+	int warningDistance = 5;
+	int warningTime = 15;
+	double centerX = 0;
+	double centerZ = 0;
```

Comment:
Missing access modifiers

Reply:
Also are these the correct default values here?

Reply:
> Also are these the correct default values here?

Yep